app/helpers/shared_helper/ab_list_helper.rb

```diff
@@ -0,0 +1,147 @@
+                          :rows  => rows
+                        })
+  end
+
+  def object_details(_custom_button, resolve)
+    data = {
+      :title => _('Object Details'),
+      :mode  => _('object_details'),
+      :rows  => [
+        row_data(_('System/Process/'), resolve[:new][:instance_name]),
+        row_data(_('Message'), resolve[:new][:object_message]),
+        row_data(_('Request'), resolve[:new][:object_request])
+      ]
+    }
+    miq_structured_list(data)
+  end
+
+  def object_attribute(custom_button, resolve)
+    custom_button&.enablement_expression
+    data = {
+      :title => _('Object Attribute'),
+      :mode  => _('object_attribute'),
+      :rows  => [
+        row_data(_('Type'), resolve[:new][:target_class]),
+      ]
+    }
+
+    if resolve[:new][:target_class].present?
+      data[:rows] << row_data(_('Selection'), resolve[:new][:target_id])
+    end
+
+    enablement_text = if custom_button&.enablement_expression.kind_of?(MiqExpression)
+                        custom_button_info_text(custom_button, :enablement_expression, custom_button&.enablement_expression&.to_human)
+                      else
+                        custom_button_info_text(custom_button, :enablement_expression, _('No Enablement Expression defined, this button will be always enabled'))
+                      end
+
+    data[:rows] << row_data(_('Enablement expression'), enablement_text)
+    data[:rows] << row_data(_('Disabled Button Text'), custom_button&.disabled_text)
+    miq_structured_list(data)
+  end
+
+  def visibility_expression(custom_button)
+    data = {
+      :title => _('Visibility Expression'),
+      :mode  => _('visibility_expression'),
+      :rows  => []
+    }
+    visibility_expression_text = if custom_button && !custom_button.visibility_expression.nil? && custom_button.visibility_expression.kind_of?(MiqExpression)
+                                   custom_button.visibility_expression.to_human
+                                 else
+                                   custom_button_info_text(custom_button, :visibility_expression, _('No Visibility Expression defined, this button will be always visible'))
+                                 end
+    data[:rows] << row_data(_('Visibility Expression'), visibility_expression_text)
+    miq_structured_list(data)
+  end
+
+  def custom_button_info_text(custom_button, attribute, default_message)
+    value = custom_button&.send(attribute)
+    value.nil? ? _(default_message) : value
+  end
+
+  def attribute_value_pair(resolve)
+    data = {
+      :title => _('Attribute/Value Pairs'),
+      :mode  => _('attribute_value_pair'),
+      :rows  => []
+    }
+    if resolve[:new][:attrs].empty?
+      data[:rows] << row_data(_('Attribute/Value Pairs'), _('No Attribute/Value Pairs found.'))
+    else
+      resolve[:new][:attrs].each_with_index do |attr, i|
+        data[:rows] << row_data(_(i + 1), [attr[0], attr[1]])
```

Comment:
`row_data((i + 1).to_s, [attr[0], attr[1]])` change this to a string.

<img width="1137" alt="image" src="https://github.com/ManageIQ/manageiq-ui-classic/assets/87487049/454bddce-dc06-459d-b946-7994910730dd">